storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/SleepTimerType.java

```diff
@@ -0,0 +1,6 @@
+package de.danoeh.antennapod.storage.preferences;
+
+public enum SleepTimerType {
```

Comment:
Move to `SleepTimer.Type`? This file is so simple, opening it is more overhead than just having it directly in the SleepTimer interface

Reply:
I tried doing that, but they're defined in different modules, we would need to add module dependencies. Is that ok?
Seems easier to just have the enum defined as it currently is.